### ISSUE TITLE
Design: Determine NAICS MVP

### ISSUE THREAD

**Issue by carye3** [newjersey/navigator.business.nj.gov#2187]
## Tasks
- [ ] Design Options
  - [ ] Include all NAICS options in Industry dropdown (in onboarding)
    - [ ] Single dropdown, or...
    - [ ] Multiple subsequent dropdowns (2 max?): high level industry, sub-industry
  - [ ] Industry dropdown inside of business formation
  - [ ] NAICS code entry field with external link to find it yourself
  - [ ] Additional task asking for NAICS prior to business formation

## Constraints

- Must be 6 digit NAICS code (per Treasury API)

## AC
- How to pass on to formation API?

**Comment by carye3** [comment]
**Ideation 3.31.22**

- NAICS: lots of possiblities. one way is for someone who we support industry for, we dont show the NAICS at all, its something treasury does.
- If we don't (they're generic) then...
we need to help them identify their NAICS code (do some research on NAICS code lookups). if thats the case we need to ask V and Neha.
- or ???

**Comment by carye3** [comment]
@aaronvux this is the story that needs to go through story writing before we can start designs

**Comment by Jesslax** [comment]
@carye3 responding below after you forwarded this from our slack convo. For the ones without answers I think we should bring to design review 
    We're still unsure if all 6 digits of the NAICS are required by Treasury for data analysis. **We should plan for level 6**  
        Initially we were in agreement to add it to onboarding, but we've since reassessed due to burden on user and possible negative impact on onboarding completion - **Think there was a miscommunication here, don't remember aligning on onboarding, but remember aligning on passing through recommendations for NAICS codes**
        Other option is adding it to formation, which we've also reassessed: this too would be onerous to user given the complexity of formation as it is (particularly if NAICS functionality was added in its most common form as a lookup table) **could we create a field for submission, repopulate it with our best guess, and beneath the field 1. offer other options as recommendations and 2. link out for them to do their own search. Similar to name search screen** 
        The last remaining option is a standalone NAICS lookup **task. also a great idea**
    How are the industry and sector information gathered in onboarding / graduation linked to NAICS? **need to discuss, I believe airtable**
    Is this is additive value to Treasury, or is there utility to the user at any point? **the user is required to find and tell treasury this information for their NJREG application. users find it hard to figure their code out, so helpful to users if we can make this easier for them**
    What digits does treasury require for data analysis vs what is necessary for the API? 
    If we make NAICS a separate task, should we make it a required dependency for starting the Formation task? **yes**
        Jess says no, that it’s optional, but it might need to be a dependency for Tax Registration instead. **I would give this a required label if it were it's own task, in slack was sharing facts about what treasury requires, but this would need to be required task**
    And if they enter the NAICS wrong in formation or tax registration, could their application be rejected? **there is no wrong answer, they will not be rejected, this is not something applications are evaluated on.**
    Can a business have multiple NAICS codes within a sector or across multiple sectors? 
    What existing information on the business can we use to narrow their NAICS results, and should we? **their industry selection in poppy**
    Have not had discussion of technical constraints for using NAICS database
    What’s an MVP in this case? 
        Should we just send users to an external NAICS database and return with that code?
        If MVP is a simple lookup table, do we need to add contextual information to all codes?
        More market research needed as NAICS lookup tables are generally bad UX particularly if you're trying to get user to determine all 6 digits
